src/app/components/PortraitVideoCarousel/index.test.tsx

```diff
@@ -12,8 +12,8 @@ describe('PortraitVideoCarousel', () => {
     });
 
     const portraitVideoItems = (container as unknown as HTMLElement)
-      .querySelector('div[data-testid="pv-scroll-panel"]')
-      ?.querySelectorAll('button');
+      .querySelector('ul[data-testid="pv-scroll-panel"]')
```

Comment:
Do we want to rename this to carousel now?
```suggestion
      .querySelector('ul[data-testid="pv-carousel"]')
```